# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium/sandbox/win/src/policy_engine_processor.h
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 2
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/chromium/sandbox/win/src/policy_engine_processor.h
--- a/security/sandbox/chromium/sandbox/win/src/policy_engine_processor.h	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/chromium/sandbox/win/src/policy_engine_processor.h	Wed Jun 12 11:10:48 2019 +0100
@@ -59,11 +59,7 @@
 //
 
 // Possible outcomes of policy evaluation.
-enum PolicyResult {
-  NO_POLICY_MATCH,
-  POLICY_MATCH,
-  POLICY_ERROR
-};
+enum PolicyResult { NO_POLICY_MATCH, POLICY_MATCH, POLICY_ERROR };
 
 // Policy evaluation flags
 // TODO(cpu): implement the options kStopOnErrors & kRankedEval.
@@ -114,8 +110,7 @@
   // created in the broker process and evaluated in the target process.
 
   // This constructor is just a variant of the previous constructor.
-  explicit PolicyProcessor(PolicyBuffer* policy)
-      : policy_(policy) {
+  explicit PolicyProcessor(PolicyBuffer* policy) : policy_(policy) {
     SetInternalState(0, EVAL_FALSE);
   }
 